test/controllers/api/v2/environments_controller_test.rb

```diff
@@ -213,7 +213,7 @@ def test_update_protected
       allowed_perms = [@update_permission]
       denied_perms = [@view_permission, @create_permission, @destroy_permission]
 
-      assert_protected_action(:destroy, allowed_perms, denied_perms, [@organization]) do
+      assert_protected_action(:update, allowed_perms, denied_perms, [@organization]) do
```

Comment:
This was the "error" I mentioned last week. The test is for the "update" method, but the tested action was "destroy".